media-gfx/pencil/pencil-0.6.4.ebuild

```diff
@@ -0,0 +1,50 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit desktop l10n qmake-utils xdg-utils
+
+DESCRIPTION="2D animation and drawing program based on Qt5"
+HOMEPAGE="https://www.pencil2d.org/"
+SRC_URI="https://github.com/pencil2d/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+PLOCALES="ca cs da de el es et fr he hu_HU id it ja kab pl pt pt_BR ru sl vi zh_CN zh_TW"
+
+RDEPEND="
+	dev-qt/qtcore:5
```

Comment:
You seem to be missing some of the required deps here. You should really study upstream build files to detect how they're used and what are unconditionally required. I see a potential for `doc` USE flag for example, then there's something for `ffmpeg` and at least qtxmlpatterns is required to build this program.